csharp/extractor/Semmle.Extraction.CSharp/Extractor/StandaloneAnalyser.cs

```diff
@@ -11,13 +13,15 @@ public StandaloneAnalyser(IProgressMonitor pm, ILogger logger, bool addAssemblyT
         {
         }
 
-        public void Initialize(CSharpCompilation compilationIn, CommonOptions options)
+        public void Initialize(string outputPath, CSharpCompilation compilationIn, CommonOptions options)
         {
             compilation = compilationIn;
-            extractor = new StandaloneExtractor(Logger, PathTransformer, options);
+            extractor = new StandaloneExtractor(outputPath, Logger, PathTransformer, options);
             this.options = options;
             LogExtractorInfo(Extraction.Extractor.Version);
             SetReferencePaths();
+
+            Entities.Compilation.Settings = (Directory.GetCurrentDirectory(), Array.Empty<string>());
```

Comment:
For consistency: Can this be put in the `Run` method for the standalone extractor like it happens in the tracing extractor?

Reply:
We can't, because the `Run` method is in `Semmle.Extraction.CSharp.Standalone` and `Compilation` is `internal`.